compiler/rustc_expand/src/mbe/macro_parser.rs

```diff
@@ -451,7 +451,7 @@ impl TtParser {
                         // Try zero matches of this sequence, by skipping over it.
                         self.cur_mps.push(MatcherPos {
                             idx: idx_first_after,
-                            matches: mp.matches.clone(), // a cheap clone
+                            matches: Lrc::clone(&mp.matches),
```

Comment:
This doesn't change behaviour, just makes the clone kind more explicit, right?

Reply:
Yes